lib/twitter_ads.rb

```diff
@@ -7,9 +7,11 @@
 #
 require 'oauth'
 require 'json'
+require 'multi_json'
+
+ADS_API_ENDPOINT = 'ads-api.twitter.com'
+TRACE = false
```

Comment:
You wouldn't normally change a constant from outside. You could call a method though, similar to `#enable_tracing` and `#disable_tracing`. That could work.

Twitter will also release their own gem in a few weeks/months. We shouldn't spend too much time on this version.